org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtBuildReporter.scala

```diff
@@ -87,7 +87,7 @@ private[zinc] class SbtBuildReporter(project: IScalaProject) extends xsbti.Repor
       val markerPos = MarkerFactory.RegionPosition(offset, identifierLength(pos.lineContent, pos.pointer), line)
       BuildProblemMarker.create(resource, severity, msg, markerPos)
     } else
-      logger.info("suppressed error in Java file: %s".format(msg))
+      logger.info(s"suppressed error in Java file ${resource.getFullPath}:$line: $msg")
```

Comment:
It is an error, why not log it as such? Do we have another logger for Java errors?